server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/http/SessionSupplier.scala

```diff
@@ -56,53 +52,26 @@ object SessionSupplier {
 }
 
 class SessionSupplier {
-  def generate(username: Username): SMono[Session] =
-    SMono.fromPublisher(
-      Mono.zip(
-        accounts(username).asJava(),
-        primaryAccounts(username).asJava()))
-      .map(tuple => generate(username, tuple.getT1, tuple.getT2))
-
-  private def accounts(username: Username): SMono[Map[Id, Account]] =
-    getId(username)
-      .map(id => Map(
-        id -> Account(
-          username,
-          IsPersonal(true),
-          IsReadOnly(false),
-          accountCapabilities = Set(CORE_CAPABILITY, MAIL_CAPABILITY))))
-
-  private def primaryAccounts(username: Username): SMono[Map[CapabilityIdentifier, Id]] =
-    SFlux.just(CORE_CAPABILITY, MAIL_CAPABILITY)
-      .flatMap(capability => getId(username)
-        .map(id => (capability.identifier, id)))
-      .collectMap(getIdentifier, getId)
-  private def getIdentifier(tuple : (CapabilityIdentifier, Id)): CapabilityIdentifier = tuple._1
-  private def getId(tuple : (CapabilityIdentifier, Id)): Id = tuple._2
-
-  private def getId(username: Username): SMono[Id] = {
-    SMono.fromCallable(() => refineId(username))
-      .flatMap {
-        case Left(errorMessage: String) => SMono.raiseError(new IllegalStateException(errorMessage))
-        case Right(id) => SMono.just(id)
-      }
+  def generate(username: Username): SMono[Session] = {
+    accounts(username)
+      .map(account => Session(
+        Capabilities(CORE_CAPABILITY, MAIL_CAPABILITY),
+        List(account),
+        primaryAccounts(account.accountId),
+        username,
+        apiUrl = new URL("http://this-url-is-hardcoded.org/jmap"),
+        downloadUrl = new URL("http://this-url-is-hardcoded.org/download"),
+        uploadUrl = new URL("http://this-url-is-hardcoded.org/upload"),
+        eventSourceUrl = new URL("http://this-url-is-hardcoded.org/eventSource"),
```

Comment:
Can we at least have a `val HARD_CODED_URL_PREFIX = http://this-url-is-hardcoded.org/`